UPGRADE.md

```diff
@@ -43,7 +43,7 @@ Follow the instructions in the [monorepo upgrade guide](upgrade/upgrading-monore
 * ### [From v8.0.1-dev to v8.1.0-dev](/upgrade/UPGRADE-v8.1.0-dev.md)
 * ### [From v8.0.0 to v8.0.1-dev](upgrade/UPGRADE-v8.0.1-dev.md)
 * ### [From v7.3.1 to v8.0.0](upgrade/UPGRADE-v8.0.0.md)
-* ### [From v7.3.1 to v7.3.2-dev](upgrade/UPGRADE-v7.3.2-dev.md)
```

Comment:
This one was actually fixed in https://github.com/shopsys/shopsys/commit/9c4116d0116bf76edded5111326da883264cb811#diff-828ad3929c2c6e3ba47c513ec3625631